src/modules/storage/BlockStorage.ts

```diff
@@ -74,30 +76,46 @@ export class BlockStorage extends Storages
                 (data.header.validators == undefined) ||
                 (data.header.validators._storage == undefined) ||
                 (data.header.merkle_root == undefined) ||
-                (data.header.signature == undefined)
+                (data.header.signature == undefined) ||
+                (data.txs == undefined)
         ) {
             if (callback != undefined)
                 callback("Parameter validation failed.");
```

Comment:
In a future PR, it would be good to say which parameter failed. And why it failed.

Reply:
In the future,  We'd better tell the customer the reason for the failure in detail.